websites/H/HBO Max/presence.ts

```diff
@@ -179,7 +170,10 @@ presence.on("UpdateData", async () => {
 					const episodeId = location.pathname.match(/:episode:([^:]+)/)[1];
 
 					if (isFetching) return;
-					coverUrls[episodeId] ??= await fetchCover();
+					await fetchCoverAndTitle();
```

Comment:
Well, you are still spamming the request

Reply:
> Well, you are still spamming the request

it only runs once i think lemme check.